strawberry/schema/types/scalar.py

```diff
@@ -16,7 +15,7 @@
 from strawberry.file_uploads.scalars import Upload
 from strawberry.scalars import ID
 
-from .base_scalars import Date, DateTime, Decimal, Time
+from .base_scalars import UUID, Date, DateTime, Decimal, Time
```

Comment:
Nitpick: did `isort` somehow not get run here?

Reply:
Weirdly I did run it and it seems to be passing now. Not sure why isort thinks that `UUID` should be at the beginning of this list.